roles/tests-integration/tests/common/mod.rs

```diff
@@ -0,0 +1,194 @@
+use std::sync::Arc;
+
+use async_std::net::TcpStream;
+use key_utils::{Secp256k1PublicKey, Secp256k1SecretKey};
+
+pub async fn start_job_declarator_server() {
+    let authority_public_key = Secp256k1PublicKey::try_from(
+        "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72".to_string(),
+    )
+    .expect("failed");
+    let authority_secret_key = Secp256k1SecretKey::try_from(
+        "mkDLTBBRxdBv998612qipDYoTK3YUrqLe8uWw7gu3iXbSrn2n".to_string(),
+    )
+    .expect("failed");
+    let cert_validity_sec = 3600;
+    let listen_jd_address = "0.0.0.0:34264".to_string();
+    let core_rpc_url = "http://75.119.150.111".to_string();
+    let core_rpc_port = 48332;
+    let core_rpc_user = "username".to_string();
+    let core_rpc_pass = "password".to_string();
+    let coinbase_outputs = vec![jd_server::CoinbaseOutput::new(
+        "P2WPKH".to_string(),
+        "036adc3bdf21e6f9a0f0fb0066bf517e5b7909ed1563d6958a10993849a7554075".to_string(),
+    )];
+    let config = jd_server::Configuration::new(
+        listen_jd_address,
+        authority_public_key,
+        authority_secret_key,
+        cert_validity_sec,
+        coinbase_outputs,
+        core_rpc_url,
+        core_rpc_port,
+        core_rpc_user,
+        core_rpc_pass,
+        std::time::Duration::from_secs(5),
+    );
+    jd_server::start_jd_server(config);
+}
+
+pub async fn start_job_declarator_client(
+) -> (jd_client::JobDeclaratorClient, jd_client::ProxyConfig) {
+    let downstream_address = "127.0.0.1".to_string();
+    let downstream_port = 34265;
+    let max_supported_version = 2;
+    let min_supported_version = 2;
+    let min_extranonce2_size = 8;
+    let withhold = false;
+    let authority_pubkey = Secp256k1PublicKey::try_from(
+        "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72".to_string(),
```

Comment:
Is there a reason we reuse keys between JDC and JDS? I would have expected them to have different keys

Reply:
that's just easy for development purposes